tools/cardano-sl-tools.cabal

```diff
@@ -17,6 +17,12 @@ Flag with-web
 
   description: Build with web part.
 
+Flag with-post-mortem-tool
+  default:     False
+  manual:      True
+
+  description: Build with web part.
```

Comment:
@gromakovsky fair enough, I don't have strong feelings about having the redundant flag there anyway. I will switch to use that and get rid of `with-post-mortem-tool`.